packages/ui/src/1_atoms/Paragraph/Paragraph.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ReactNode, useMemo } from 'react';
+
+import classNames from 'classnames';
+
+import styles from './Paragraph.module.css';
+import { ParagraphSize, ParagraphStyle } from './Paragraph.types';
+
+export type ParagraphProps = {
+  children: ReactNode;
+  size?: ParagraphSize;
+  style?: ParagraphStyle;
+  className?: string;
+};
+
+export const Paragraph: React.FC<ParagraphProps> = ({
+  children,
+  size = ParagraphSize.base,
+  style = ParagraphStyle.normal,
+  className,
+}) => {
+  const classNameComplete = useMemo(
```

Comment:
It's not necessary to use `useMemo` in this case - apply all of the classes directly and it will simplify the component.